Cache.ts

```diff
@@ -165,12 +181,14 @@ class Cache {
     }
 
     // Returns the number of non-expired keys in the cache
-    get count() {
+    get count(options: CountOptions = { invalidateExpired: false }) {
         let ret = 0;
 
         this._cache.forEach((data, key) => {
             if (!data.expired) {
                 ret++;
+            } else if (options.invalidateExpired) {
```

Comment:
This check doesn't need to be done for each element, should be placed outside the forEach function